Helpers/VesselHelper.cs

```diff
@@ -74,6 +74,12 @@ public static class VesselHelper {
           if (Module.moduleValues.GetValue ("name") == module) {
             return true;
           }
+		  //if the module being searched for is ModuleDataTransmitter (stock antenna), and that module was not found, now look for ModuleRTAntenna, the RemoteTech Antenna Module.
```

Comment:
Gotcha. Let me clean this up a bit by adding a aliases so we don't end up with hundreds of edge cases and I'll merge this in. Thanks!